prody/proteins/mmtffile.py

```diff
@@ -221,7 +223,7 @@ def _bio_transform(dec):
         ret[t['name']] = L
     return ret
 
-def set_info(atomgroup, mmtf_data,get_bonds=False,altloc_sel='A'):
+def set_info(atomgroup, mmtf_data,get_bonds=False,altloc_sel=None):
```

Comment:
I wouldn't use None here because that has a defined behaviour in the rest of ProDy of handling altlocs in a completely different way of making multiple coordinate sets as I mentioned earlier (see #1783 where I fixed some bugs with that).

I also don't see the point of setting it to None and then resetting it to 'A' if it's None.

Reply:
You may want to consider keeping all entries with no altloc too. That's what happens for mmCIF if which_altloc=None:
https://github.com/prody/ProDy/blob/cdbed98fb33d6073bc23491549c54c7c5fb87db9/prody/proteins/ciffile.py#L364

Reply:
The point of setting it to None is to indicate that it was not set by the user, which I can't tell if it is set to A by default.  The idea being if it is one of the 60 PDBs where every atom has an altloc specified and the user asks for 'A' and there is no 'A' (which is only true for 6uwi) they should get an empty molecule back.

Reply:
Good point 

The case where None gave multiple coordinate sets was surprising anyway